### ISSUE TITLE
Problem with htaccesstest.txt and setup.js after a docker installation

### ISSUE THREAD

**Issue by floatingpurr** [nextcloud/docker#1231]
Hello there,

I'm trying to do a fresh docker installation of Nextcloud 19.0.2. Here is my compose file

   
```yml
 version: '3.8'

    services:
      # The main app
      nextcloud:
        image: nextcloud
        networks:
          - default
          - proxy
        volumes:
          # Main folder, needed for updating
          - html:/var/www/html
          # installed / modified apps
          - custom_apps:/var/www/html/custom_apps
          # local configuration
          - config:/var/www/html/config
          # actual data of your Nextcloud!!!!
          - data:/var/www/html/data
          # Themes and branding
          - themes:/var/www/html/themes/
        restart: always
        depends_on:
          - db
          - redis
        environment:
          - POSTGRES_HOST=${POSTGRES_HOST}
          - POSTGRES_DB=${POSTGRES_DB}
          - POSTGRES_USER=${POSTGRES_USER}
          - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
          # - NEXTCLOUD_ADMIN_USER=${NEXTCLOUD_ADMIN_USER}
          # - NEXTCLOUD_ADMIN_PASSWORD=${NEXTCLOUD_ADMIN_PASSWORD}
          - REDIS_HOST=${REDIS_HOST}
          - REDIS_HOST_PORT=${REDIS_HOST_PORT}
          - REDIS_HOST_PASSWORD=${REDIS_HOST_PASSWORD}
          - NEXTCLOUD_TRUSTED_DOMAINS=nextcloud.example.org
          - OVERWRITEPROTOCOL=https
          
        labels:
          traefik.enable: true
          traefik.http.routers.nextcloud.rule: Host(`nextcloud.example.org`)
          traefik.http.routers.nextcloud.entrypoints: websecure
          traefik.http.routers.nextcloud.tls.certresolver: tlsleresolver
          traefik.http.routers.nextcloud.middlewares: nextcloud-redirectregex,nc-header

          # Custom middleware for Cal/Cardav routing
          traefik.http.middlewares.nextcloud-redirectregex.redirectRegex.permanent: true
          traefik.http.middlewares.nextcloud-redirectregex.redirectRegex.regex: https://(.*)/.well-known/(card|cal)dav
          traefik.http.middlewares.nextcloud-redirectregex.redirectRegex.replacement: https://$${1}/remote.php/dav/

          # Custom middleware for headers
          traefik.http.middlewares.nc-header.headers.stsSeconds: 15552001
     
      # nextcloud is backed by postgres
      db:
        image: postgres
        networks:
          - default
        restart: always
        volumes:
          - dbdata:/var/lib/postgresql/data/
        environment:
          - POSTGRES_HOST=${POSTGRES_HOST}
          - POSTGRES_DB=${POSTGRES_DB}
          - POSTGRES_USER=${POSTGRES_USER}
          - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}


      # Redis is used to prevent file locking problems
      redis:
        image: redis:alpine
        networks:
          - default
        command: redis-server --requirepass ${REDIS_HOST_PASSWORD}
        volumes:
          - redisdata:/data


    volumes:
      html:
      custom_apps:
      config:
      data:
      themes:
      dbdata:
      redisdata:


    networks:
      proxy:
        external: true
````

All seems going fine but in the App log I see the following recurring errors:

> [no app in context] Warning: Could not detect any host in https:///data/htaccesstest.txt
> 
> GET /
> from xxx.xxx.xxx.xxx at 2020-08-31T14:21:16+00:00

> [jsresourceloader] Error: Could not find resource js/setup.js to load
> 
> GET /
> from xxx.xxx.xxx.xxx at 2020-08-31T14:21:16+00:00

How can I fix those problems?

**Comment by Issen007** [comment]
Did you solve this?

**Comment by floatingpurr** [comment]
Nope :/

**Comment by Issen007** [comment]
Found out that this problem has no impact on the nextcloud installation.
But if you want to erase the message you can always create the `setup.js` file under `./core/js/` directory

**Comment by joshtrichards** [comment]
Duplicate of nextcloud/server#27575
Likely fixed in nextcloud/server#28995 and elsewhere

In any case, not a Docker image matter. And this issue is 3 years old so I'm closing. :)
